ui/test/nuxeo-date-picker.test.js

```diff
@@ -97,5 +116,26 @@ suite('nuxeo-date-picker', () => {
         expect(localEltValue.second()).to.be.equal(19);
       });
     });
+
+    suite(!conf.timezone ? 'locale with no timezone' : `locale with ${conf.timezone} timezone`, () => {
+      let currentLocale;
+
+      setup(async () => {
+        currentLocale = moment.locale();
```

Comment:
Was just wondering if we do in fact need a dedicated suite. 🤔  Couldn't we do this already on the suite above and on the teardown do the restore only if the locale changed?

Reply:
🤔 well, maybe we don't even need to check if it changed. We could keep a single suite, and on the teardown, after each test it restores the locale (it wouldn't hurt to restore it, even if it hasn't changed). Wdyt?